rules/solve_electricity.smk

```diff
@@ -4,6 +4,18 @@
 
 
 rule solve_network:
+    params:
+        solving=config["solving"],
+        config_parts={
```

Comment:
can we define the dictionary in the script instead? and use direct assignments here instead?